inception-imls-opennlp/src/test/resources/de/tudarmstadt/ukp/dkpro/core/api/datasets/lib/sentence-classification-en.yaml

```diff
@@ -0,0 +1,56 @@
+groupId: org.dkpro.core.datasets.sentenceclassification
```

Comment:
Why do we need this file？

Reply:
It is used to download the data used in the unit test.